pinecone/client.go

```diff
@@ -543,7 +1157,7 @@ func buildClientBaseOptions(in NewClientBaseParams) []control.ClientOption {
 	return clientOptions
 }
 
-func ensureURLScheme(inputURL string) (string, error) {
+func ensureURLSchema(inputURL string) (string, error) {
```

Comment:
@austin-denoble I changed the name of this function while I was updating the documentation for this file, since [I think it should be](https://helpcenter.trendmicro.com/en-us/article/tmka-18277#:~:text=URL%20schema%20is%20used%20as,are%20usually%20http%20or%20https.) "schemA", not "schemE", but correct me if I'm wrong!